src/Html/Inert.elm

```diff
@@ -46,6 +45,11 @@ impossibleMessage =
     "An Inert Node fired an event handler. This should never happen! Please report this bug."
 
 
+attributeName : Html.Attribute a -> String
+attributeName attribute =
+    Native.HtmlAsJson.attributeName attribute
```

Comment:
@lukewestby :thought_balloon: there's no need for this function to be native in this way. I'd pass it to our stringify function, then use a decoder. It's more trust worthy, and allows us to better represent this as an AST